<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>315</width>
      <height>360</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPlayer">
         <property name="text">
          <string>Current Player</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonShow">
         <property name="text">
          <string>&amp;Show Player</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSlider" name="positionSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonKMix">
         <property name="text">
          <string>Show KMi&amp;x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxMute">
         <property name="text">
          <string>&amp;Mute</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string>Sorry,</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelArtist">
             <property name="text">
              <string>Unable to fetch data</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelAlbum">
             <property name="text">
              <string>from the music player</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horButtonLayout">
             <item>
              <widget class="QPushButton" name="pushButtonPrev">
               <property name="maximumSize">
                <size>
                 <width>73</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&amp;Previous</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonPlay">
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Play/Pa&amp;use</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonNext">
               <property name="maximumSize">
                <size>
                 <width>54</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&amp;Next</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelArt">
           <property name="maximumSize">
            <size>
             <width>111</width>
             <height>111</height>
            </size>
           </property>
           <property name="text">
            <string>NoImage</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QSlider" name="volumeSlider">
             <property name="toolTip">
              <string extracomment="Volume Slider"/>
             </property>
             <property name="statusTip">
              <string extracomment="Volume Slider"/>
             </property>
             <property name="whatsThis">
              <string extracomment="Volume Slider"/>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxMute_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxMute</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBoxMute</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMute</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBoxMute_2</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMute_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkBoxMute</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
